bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss

```diff
@@ -30,3 +30,7 @@
 .invisible {
   visibility: hidden;
 }
+
+.isTalking {
+  text-shadow: 0 0 .35rem #FFF;
```

Comment:
Should'nt it be `box-shadow` instead of `text-shadow`? Or the ideia is to have only the icon glowing and not the whole button?